app/models/application_record.rb

```diff
@@ -1,8 +1,7 @@
 include ApplicationConfiguration
 
 class ApplicationRecord < ActiveRecord::Base
-  belongs_to :user
-  belongs_to :position
+  belongs_to :user, :position
```

Comment:
See [the build failure](https://travis-ci.org/umts/jobapps/builds/129153664) - I don't think this works.